testdata/p4_16_samples_outputs/psa-example-select_tuple-wc.p4

```diff
@@ -1,5 +1,5 @@
 #include <core.p4>
-#include <bmv2/psa.p4>
```

Comment:
Are these changes because the file is DPDK-related?

Reply:
testdata/p4_16_samples/psa-example-select_tuple-wc.p4 was added in a dpdk-related PR (#2836), so the intent was likely to include dpdk/psa.p4. However, testdata/p4_16_samples/psa-example-select_tuple-wc.p4 contained `#include <psa.p4>`, so bmv2/psa.p4 was actually included.